tests/test_worker/test_categorization.py

```diff
@@ -9,7 +9,7 @@ def test_categorize_item(self, categorizer, ebolaItem):
         overlapping_categories = filter(lambda x: "Ebola" in x or "West Africa" in x, categories)
```

Comment:
Add comment: makes call to Wikipedia API (so we know that an internet connection is required)